meta-mender-nxp/README.md

```diff
@@ -2,9 +2,9 @@
 
 Supported boards:
 
- - WaRP7
- - Pico-Pi i.MX7D
- - Nitrogen8M
+ - [WaRP7](https://hub.mender.io/t/nxp-warp7/135)
+ - [Pico-Pi i.MX7D](https://hub.mender.io/t/technexion-pico-pi-imx7/136)
+ - [Nitrogen8M](https://hub.mender.io/t/boundary-devices-nitrogen8m/409)
 
 ## Build
```

Comment:
Rename `## Build` to `## Quick start` for consistency.

Reply:
Done ;)